# Subscriber (LiDAR) Params
pointcloud_topic: "/points_raw" 

# Publisher (BBOX) Parmas
bbox_topic: "/detect_3dbox"
threshold: 0.1

# OpenPCDet Params
#cfg_file: "OpenPCDet/tools/cfgs/kitti_models/pv_rcnn.yaml"
#ckpt_file: "OpenPCDet/ckpt/pv_rcnn_8369.pth" 
#
#cfg_file: "OpenPCDet/tools/cfgs/kitti_models/voxel_rcnn_car.yaml"
#ckpt_file: "OpenPCDet/ckpt/voxel_rcnn_car_84.54.pth" 
#
#cfg_file: "OpenPCDet/tools/cfgs/kitti_models/second.yaml"
#ckpt_file: "OpenPCDet/ckpt/second_7862.pth" 
#
cfg_file: "OpenPCDet/tools/cfgs/kitti_models/second_iou.yaml"
ckpt_file: "OpenPCDet/ckpt/second_iou7909.pth" 
#
#cfg_file: "OpenPCDet/tools/cfgs/kitti_models/PartA2.yaml"
#ckpt_file: "OpenPCDet/ckpt/PartA2_7940.pth" 
